spec/features/lesson/listing_lessons_spec.rb

```diff
@@ -69,6 +70,22 @@
     expect(data.size).to eq(1)
   end
 
+  it 'does not include lessons from outside the class if school_class_id provided' do
+    get("/api/lessons?school_class_id=#{school_class.id}", headers:)
+    data = JSON.parse(response.body, symbolize_names: true)
```

Comment:
I'd feel more confident in this step if there was a lesson associated with a school_class that isn't being returned, but it doesn't look like there is?

Reply:
Thanks Dan, added another test for this 👍